indico/cli/core.py

```diff
@@ -146,6 +144,7 @@ def cleanup(temp, cache, assets, verbose, dry_run, min_age):
 @click.option('--evalex-from', multiple=True,
               help='Restrict the debugger shell to the given ips (can be used multiple times)')
 @click.option('--proxy', is_flag=True, help='Use the ip and protocol provided by the proxy.')
+@click.option('--reloader-type', default='auto', help='Use specific reloader type for your devserver.')
```

Comment:
I'd use `type=click.Choice(...)` so people actually know what they can use. And maybe rename the option to `--reloader`